packages/applications-service-api/src/models/attachment.js

```diff
@@ -4,10 +4,10 @@ module.exports = (sequelize, DataTypes) => {
 	class Attachment extends Model {
 		static async findAllAttachmentsWithCase(caseReference, options = {}) {
 			const tableName = Attachment.getTableName();
-			const attachmentsSql = sequelize.dialect.QueryGenerator.selectQuery(
+			const attachmentsSql = sequelize.dialect.queryGenerator.selectQuery(
```

Comment:
QueryGenerator is now lowercase from version 6

https://sequelize.org/docs/v6/other-topics/upgrade/